css-brochure-form/styles.css

```diff
@@ -0,0 +1,63 @@
+form {
+  width: 50%;
+  margin: auto;
+  margin-left: 6.8in;
+}
+
+p {
+  padding-left: 10px;
+  margin-bottom: 6px;
+  font-family: BrandonGrotesque, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+input {
+  margin: 4px;
+  width: 53%;
+  padding: 10px;
+  border-style: solid;
+  border: 1px solid rgb(226 226 226);
+  border-radius: 5px;
+  font-size: 17px;
+  font-family: AtlasGrotesk, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  flex
+}
+
+h1 {
+  text-align: center;
+  color: rgb(211 89 71);
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: bold;
+  font-weight: 300;
+  font-size: 20px;
+}
+
+select {
+  margin: fixed;
+  width: 458px;
```

Comment:
Why the pixel width? If you set your `form` to the right width, you should be able to use `100%`